test/Project.toml

```diff
@@ -4,7 +4,6 @@ LazyArtifacts = "4af54fe1-eca0-43a8-85a7-787d91b784e3"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-TestItemRunner = "f8b46487-2199-4994-9208-9a1283c18c0a"
```

Comment:
ah, should delete the `TestItemRunner` in the `Project.toml` in the root folder, this one in the `test/Project.toml` should be kept.

Reply:
Yes, so sorry, missclick